projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/index.tsx

```diff
@@ -37,22 +47,19 @@ const BackupUpgrade = ( { comments, isFetchingData, plugins, posts } ) => {
 		setShowPopup( false );
 	}, [ comments, plugins, posts ] );
 
-	return (
-		! isFetchingData &&
-		showPopup &&
-		posts > MIN_POSTS_FOR_VISIBLE_BAR && (
-			<BarChart
-				posts={ posts }
-				comments={ comments }
-				plugins={ plugins }
-				onClosePopup={ onClosePopup }
-			/>
-		)
-	);
+	return ! isFetchingData && showPopup && posts > MIN_POSTS_FOR_VISIBLE_BAR ? (
+		<BarChart
+			posts={ posts }
+			comments={ comments }
+			plugins={ plugins }
+			onClosePopup={ onClosePopup }
+		/>
+	) : null;
 };
 
 export default connect( state => {
-	const stats = getStatsData( state )?.general?.stats;
+	// eslint-disable-next-line  @typescript-eslint/no-explicit-any
+	const stats = ( getStatsData( state ) as any )?.general?.stats;
```

Comment:
It's because `getStatsData` is not (yet) properly typed and return type is set to `Object` which TS doesn't like if you try to access its properties because those props don't exist for TS.

Reply:
@manzoorwanijk but you don't have the object shape to create the interface for that?

Reply:
No, I have no idea what that object looks like.